<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="output">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>321</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Eingabe</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numIn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="nameIn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="nameReq">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Absenden</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reqButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Request</string>
    </property>
   </widget>
   <widget class="QLabel" name="numOut">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Nummer</string>
    </property>
   </widget>
   <widget class="QLabel" name="light">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
